task-1/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Task-1</title>
+</head>
+<body>
+
+    <a href="../index.html">Back to home</a>
+
+    <!-- Scripts -->
+    <script src="../js/script.js"></script>
```

Comment:
Place this script inside `task-1-1` folder. Let's keep the structure scoped by tasks. We're not building a regular website, just learning.